### DISCUSSION TITLE
Can't get Panda to output pattern styles on my repo

### DISCUSSION THREAD

**Discussion by osdiab** [chakra-ui/panda#867] [Q&A] [answered]
Thought it was a general issue about monorepos at https://github.com/chakra-ui/panda/discussions/847 but turns out I had misconfigured that repro case.

But I'm still not sure why when I try to do this in another test project that already uses vanilla-extract that it seems to fail.

https://github.com/osdiab/personal-website/pull/11

So basically i get this markup:

<img width="497" alt="Screenshot 2023-06-28 at 14 44 55" src="https://github.com/chakra-ui/panda/assets/1890926/4aa8fefa-232d-4072-8f26-5e179502d606">

but the `hstack` fails to show up like so:

<img width="437" alt="Screenshot 2023-06-28 at 14 44 48" src="https://github.com/chakra-ui/panda/assets/1890926/e999965d-828f-44c6-b972-c1b935f93c91">

and even eradicating all traces of vanilla extract doesn't help: https://github.com/osdiab/personal-website/pull/12

<img width="157" alt="Screenshot 2023-06-28 at 14 54 11" src="https://github.com/chakra-ui/panda/assets/1890926/3bf6fea5-e852-40b1-923b-65d4bae9a3f2">

At a loss for what I'm doing wrong and would appreciate some guidance if anyone might know where I've turned astray. Thanks!

**Comment by osdiab** [comment, marked as answer]
I think I figured it out, I was using TypeScript paths to alias `src/*` to `@/*` which NextJS natively supports, but doing so apparently breaks Panda. If I refer to it using not the alias but instead using a relative path, then it works. Looking into see if there's a way to get around that issue.

**Reply by anubra266** [reply]
@osdiab do you mean you were doing this in your panda config?

**Reply by astahmer** [reply]
related https://github.com/chakra-ui/panda/issues/868
